src/uikit.cs

```diff
@@ -3253,12 +3282,24 @@ interface UIContextMenuInteraction : UIInteraction {
 		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
 		NSObject WeakDelegate { get; }
 
+		[iOS (14,0)]
+		[Export ("menuAppearance")]
+		UIContextMenuInteractionAppearance MenuAppearance { get; }
+
 		[Export ("initWithDelegate:")]
 		[DesignatedInitializer]
 		IntPtr Constructor (IUIContextMenuInteractionDelegate @delegate);
 
 		[Export ("locationInView:")]
 		CGPoint GetLocation ([NullAllowed] UIView inView);
+
+		[iOS (14,0)]
+		[Export ("updateVisibleMenuWithBlock:")]
+		void UpdateVisibleMenu (Func<UIMenu, UIMenu> handler);
```

Comment:
Shall we use a delegate with names of what UIMenu first and UIMenu second are?

Reply:
We could but we gain not much from that, what you get in the parameter is the visible menu and the method is already called Update`VisibleMenu` and the return is well a menu too :/